packages/block-editor/src/components/block-list/block.js

```diff
@@ -49,7 +49,6 @@ import BlockInsertionPoint from './insertion-point';
 import IgnoreNestedEvents from '../ignore-nested-events';
 import InserterWithShortcuts from '../inserter-with-shortcuts';
 import Inserter from '../inserter';
-import useHoveredArea from './hover-area';
```

Comment:
This hook is not used anymore, right? Can we remove it?

Reply:
You're right, good call! Removed in https://github.com/WordPress/gutenberg/pull/17315/commits/40fbc51e7d836be566c5768a6bbd21eb0aa5ced4